internal/rest/avi_obj_vsvip.go

```diff
@@ -491,16 +504,37 @@ func (rest *RestOperations) AviVsVipCacheAdd(rest_op *utils.RestOp, vsKey avicac
 			if found {
 				vs_cache_obj.AddToVSVipKeyCollection(k)
 				utils.AviLog.Debugf("key: %s, msg: modified the VS cache object for VSVIP collection. The cache now is :%v", key, utils.Stringify(vs_cache_obj))
+				vip := vsvip_cache_obj.Vips[0]
```

Comment:
I think we need a length check here for Vips like its being done for fips.

Reply:
added a length check